app/controllers/tag_controller.rb

```diff
@@ -279,4 +279,9 @@ def remove_all_tags
   def location
     render template: 'locations/_form'
   end
+
+  def embed
+      render layout: false
```

Comment:
Good start! Here, if we want to use the "gridsEmbed.html.erb" template, we'll have to specify it with `template: "gridsEmbed"` or name the action to match the template file: `def gridEmbed` (would be great for it to be singular, as it makes a bit more sense -- you'll have to change the template file too).